### DISCUSSION TITLE
FFXIII doesn't start with the patch installed

### DISCUSSION THREAD

**Discussion by KaiMayflower23** [rebtd7/FF13Fix#107] [Q&A]
hello all, my FFXIII doesn't launch with the patch installed, removing the patch it launch so hmm idk what to do.. playing without the patch is bad u.u  btw in 2022 i have played the game with the patch and nothing wrong happens but now i cant, anyone know why happen this?

**Comment by hypyxo** [comment]
Mine allows FF13Fix.ini to stay but will not run with the d3d9.dll. I am not experienced enough to know the dependency on this mod working's reliance on this Direct X file but hoping the .ini file does the heavy lifting. I've not found a way to have both files installed. I have also noted how a third file "FF13Fix.log" gets created too in the game folder, which also is fine to stay apparently.
